tests/test_stream_daq.py

```diff
@@ -113,6 +121,37 @@ def test_csv_output(tmp_path, default_streamdaq, write_metadata, caplog):
         default_streamdaq.capture(source="fpga", metadata=None, show_video=False)
         assert not output_csv.exists()
 
+def capture_wrapper(default_streamdaq, source, show_video, continuous):
+    try:
+        default_streamdaq.capture(source=source, show_video=show_video, continuous=continuous)
+    except KeyboardInterrupt:
+        print("KeyboardInterrupt caught as expected")
+
+@pytest.mark.skip("Temporary skipped because tests fail in some OS (See GH actions).")
+@pytest.mark.timeout(10)
+def test_continuous_and_termination(tmp_path, default_streamdaq):
+    """
+    Make sure continuous mode runs forever until interrupted, and that all processes are
+    cleaned up when the capture process is terminated.
+    """
+    timeout = 5
+
+    capture_process = multiprocessing.Process(target=capture_wrapper, args=(default_streamdaq, "fpga", False, True))
+
+    capture_process.start()
+    alive_processes = default_streamdaq.alive_processes()
+    initial_alive_processes = len(alive_processes)
+    
+    time.sleep(timeout)
+
+    alive_processes = default_streamdaq.alive_processes()
+    assert len(alive_processes) == initial_alive_processes
+    
+    os.kill(capture_process.pid, signal.SIGINT)
+    capture_process.join()
+
+    alive_processes = default_streamdaq.alive_processes()
+    assert len(alive_processes) == 0
```

Comment:
Here's the skipped test block. It basically:
1. Starts processes
2. Counts initial processes
3. Counts after a while to check if continuous mode is working
4. Counts after terminating to check if it's zero

And 4 doesn't work in some environments.